src/pages/attribute-dictionary.js

```diff
@@ -246,7 +246,7 @@ const EventDefinition = memo(({ location, event, filteredAttribute }) => {
             margin-right: 0.5rem;
           `}
         >
-          Data {pluralize('source', event.dataSources.length)}
+          Data {(pluralize('source', event.dataSources.length), ':')}
```

Comment:
```suggestion
          Data {(pluralize('source', event.dataSources.length))}:
```

Does what you have work? I can't say I've tried it with parens like that 😄 . In case it doesn't, the above suggestion should do it.

Reply:
I did work, but I trust your code more than mine. lol